analytics.js

```diff
@@ -60,7 +60,7 @@ function enqueue(rudderElement, type) {
   if (!this.eventRepository) {
     this.eventRepository = EventRepository;
   }
-  this.eventRepository.enqueue(rudderElement, type);
+  this.eventRepository.enqueue(rudderElement, type, this.beaconPlugin);
```

Comment:
?

Reply:
By mistake, removed it

Reply:
Appreciate the quick response but can you respond after you push the changes? I can re-verify and resolve it.